content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,243 +5,255 @@ weight = 20
       </tr>
       <tr>
-        <td>Kubeflow on Azure</td>
-        <td>Microsoft Azure</td> 
         <td>
-          Azure Kubernetes Service (AKS)
+          All Certified Kubernetes Distributions
         </td>
         <td>
-          <a href="https://azure.github.io/kubeflow-aks/main">Website</a>
+          {{% deploykf/latest-version %}} <sup>[<a href="https://www.deploykf.org/releases/version-matrix/#kubeflow-tools">Version Matrix</a>]</sup>
+        </td>
+        <td>
+          Aranui Solutions / deployKF
         </td>
         <td>
-          1.7.0 <sup>[<a href="https://github.com/Azure/kubeflow-aks/releases/tag/v1.7.0">Release Notes</a>]</sup>
+          <a href="https://www.deploykf.org/">Website</a>
         </td>
       </tr>
       <tr>
-      <tr>
-        <td>Kubeflow on Google Cloud</td>
-        <td>Google Cloud</td>
         <td>
           Google Kubernetes Engine (GKE)
         </td>
         <td>
-          <a href="https://googlecloudplatform.github.io/kubeflow-gke-docs">Website</a>
+          {{% gke/latest-version %}} <sup>[<a href="https://googlecloudplatform.github.io/kubeflow-gke-docs/docs/changelog/">Release Notes</a>]</sup>
         </td>
         <td>
-          1.8.0 <sup>[<a href="https://googlecloudplatform.github.io/kubeflow-gke-docs/docs/changelog/#180">Release Notes</a>]</sup>
+          Google Cloud
+        </td>
+        <td>
+          <a href="https://googlecloudplatform.github.io/kubeflow-gke-docs">Website</a>
         </td>
       </tr>
       <tr>
-        <td>Kubeflow on IBM Cloud</td>
-        <td>IBM Cloud</td>
         <td>
           IBM Cloud Kubernetes Service (IKS)
         </td>
         <td>
-          <a href="https://ibm.github.io/manifests/">Website</a>
+          {{% iks/latest-version %}} <sup>[<a href="https://github.com/IBM/manifests/releases/tag/v{{% iks/latest-version %}}">Release Notes</a>]</sup>
         </td>
         <td>
-          {{% iks/latest-version %}} <sup>[<a href="https://github.com/IBM/manifests/releases/tag/v{{% iks/latest-version %}}">Release Notes</a>]</sup>
+          IBM Cloud
+        </td>
+        <td>
+          <a href="https://ibm.github.io/manifests/">Website</a>
         </td>
       </tr>
       <tr>
-        <td>Kubeflow on Nutanix</td>
-        <td>Nutanix</td>
         <td>
           Nutanix Kubernetes Engine
         </td>
         <td>
-          <a href="https://nutanix.github.io/kubeflow-manifests">Website</a>
+          {{% nutanix/latest-version %}}
         </td>
         <td>
-          {{% nutanix/latest-version %}}
+          Nutanix
+        </td>
+        <td>
+          <a href="https://nutanix.github.io/kubeflow-manifests">Website</a>
         </td>
       </tr>
       <tr>
-        <td>Charmed Kubeflow</td>
-        <td>Canonical</td>
         <td>
-          All Certified Kubernetes Distributions <sup>[<a href="https://kubernetes.io/partners/#conformance">1</a>]</sup>
+          Amazon Elastic Kubernetes Service (EKS)
```

Comment:
I still think we should consider removing this distribution, AWS has indicated they are no longer supporting it, so it's a bit crazy to include it (which will make people adopt a distribution that will not get any updates):

Here is their announcement that they will not support 1.8:

- https://github.com/awslabs/kubeflow-manifests/issues/794
- And looking by [commits on the repo](https://github.com/awslabs/kubeflow-manifests) I am pretty sure they are not maintaining it.

Reply:
@ananth102 you're the only person to commit on the `awslabs/kubeflow-manifests` repo in the last 5 months, what is the status of that repo?

Reply:
I have different thoughts here.
Since EKS is one of the most popular Kubernetes environment and many folks are considering to use Kubeflow distribution that was created and was maintained by AWS folks, we might want to keep it for a while.
Even that AWS was not participating in the recent Kubeflow release.
In the future, AWS might work on this again.

Also, as I said here: https://github.com/kubeflow/website/pull/3693#issuecomment-1976597192, we should list distributions with `release-7` support.

cc @kubeflow/kubeflow-steering-committee @surajkota @jsitu777 @ryansteakley